src/main/12-cognito-lambda-msg.tf

```diff
@@ -25,10 +25,21 @@ resource "aws_iam_role" "lambda_cognito_messaging_execution_role" {
   assume_role_policy = data.aws_iam_policy_document.congito_messaging_assume_role.json
 }
 
+resource "null_resource" "lambda_cognito_messaging" {
+  provisioner "local-exec" {
+    command = "cd ${path.module}/assets/lambda_cognito_messaging/ && npm install"
+  }
+
+  triggers = {
+    always_run = "${timestamp()}"
+  }
```

Comment:
Use same strategy as lambda authorizers

Reply:
Can't use because in subsequent apply node_modules are note installed. The zipfile is built empty if not triggered